unified-runtime/source/adapters/level_zero/usm.cpp

```diff
@@ -949,7 +949,21 @@ ur_usm_pool_handle_t_::ur_usm_pool_handle_t_(ur_context_handle_t Context,
     }
   }
 
-  auto [Ret, Descriptors] = usm::pool_descriptor::create(this, Context);
+  std::vector<ur_device_handle_t> DevicesAndSubDevices;
+  std::function<void(std::vector<ur_device_handle_t> &)>
```

Comment:
could you please implement this function in some common file (shared between v1 and v2)? I think we should use the same approach in v2.

Reply:
Done.